pytorch_lightning/plugins/environments/slurm_environment.py

```diff
@@ -62,18 +65,18 @@ def master_port(self):
 
         log.debug(f"MASTER_PORT: {os.environ['MASTER_PORT']}")
 
-        return default_port
+        return int(default_port)
 
     def world_size(self):
```

Comment:
```suggestion
    def world_size(self) -> Optional[int]:
```

Reply:
Why SLURM cluster is returning None now for world_size ?

Reply:
it was referencing `self._world_size` which was always None anyway :)